### ISSUE TITLE
-passthru doesn't work anymore

### ISSUE THREAD

**Issue by magnumripper** [magnumripper/jtrTestSuite#54]
WTF happened?
```
$ ./jtrts.pl -internal -passthru=-dev=0 opencl 
-------------------------------------------------------------------------------
- JtR-TestSuite (jtrts). Version 1.13, Dec 21, 2014.  By, Jim Fougeron & others
- Testing:  John the Ripper 1.8.0.6-jumbo-1-352-g44c92c6 OMP [darwin14.4.0 64-bit AVX-ac]
--------------------------------------------------------------------------------

John Jumbo build detected.
stat: v=0: No such file or directory
Running JTRTS in -internal mode
NO tests were performed.  Time used was 1 seconds
```

**Comment by magnumripper** [comment]
Looks like it could be caused by f3d889a. Those are very very promiscuous regexps, LOL...

**Comment by frank-dittrich** [comment]
Fixed with commit ed08574981bbcd8472bcdaa7e524afa92e286004 and commit 0dd7269203bdbd7a9a1d9c54a8ccb4b3113f5bc7.
The regexps were OK for non-GPU builds, but not for GPU builds.
